tests/conftest.py

```diff
@@ -41,7 +42,7 @@ def params(ts_stats, tmp_path):
 def era5_hr():
     import netCDF4 as nc
     """Return a netCDF file with hourly ERA5 data at the Salmon location."""
-    path = TD / "ERA5" / "ts.nc"
+    path = Path(TD) / "ERA5" / "ts.nc"
```

Comment:
TD should already be a Path instance.

Reply:
True. I wasn't sure why Path was imported. Given that TD is a class instance, I suppose there isn't a need for that library then.